lang/semgrep-grammars/src/semgrep-typescript/common/semgrep-ext.js

```diff
@@ -39,6 +39,34 @@ module.exports = {
       $.optional_parameter,
     ),
 
+    class_body: $ => seq(
```

Comment:
At least add a big coupling comment here saying that each
time we update the grammar we should make sure we update this class_body to reflect changes
in the original grammar.

Reply:
Make sense. I've added the comment.